crates/wasmlink/build.rs

```diff
@@ -0,0 +1,32 @@
+use std::process::{Command, Stdio};
+
+fn main() {
+    println!("cargo:rerun-if-changed=runtime/src");
+    println!("cargo:rerun-if-changed=runtime/Cargo.toml");
+    println!("cargo:rerun-if-changed=runtime/Cargo.lock");
+
+    let mut cmd = Command::new("cargo");
+
+    cmd.args(&["build", "--target=wasm32-wasi"]);
+
+    if !cfg!(debug_assertions) {
+        cmd.arg("--release");
+        println!("cargo:rustc-env=RUNTIME_WASM_PATH=../runtime/target/wasm32-wasi/release/wasmlink_runtime.wasm");
```

Comment:
For release mode this may want to use `CARGO_PROFILE_RELEASE_LTO=yes` to force-on that? Along the lines of producing a lean-as-possible module

Reply:
I'll add that, thanks.